scripts/transform_script/src/transform.py

```diff
@@ -0,0 +1,124 @@
+import argparse
+import os
+from PIL import Image
+import uuid
+from argparse import Namespace
+from datetime import datetime
+from io import BytesIO
+from typing import List
+
+from minio import Minio
+
+# python3 src/transform.py Labrador raw-images transformed-images 2022-01-20 254 254
+
+def parse_date(date: str) -> datetime:
+    parsed_date = datetime.strptime(date, "%Y-%m-%d")
+
+    return parsed_date
+
+
+def cli() -> Namespace:
+    parser = argparse.ArgumentParser()
+    parser.add_argument('search_request', type=str, help='help')
+    parser.add_argument('source_bucket_name', type=str, help='help')
+    parser.add_argument('load_bucket_name', type=str, help='help')
+    parser.add_argument('date', type=parse_date, help='help')
+    parser.add_argument('width', type=int, help='help')
+    parser.add_argument('height', type=int, help='help')
+
+    arguments = parser.parse_args()
+
+    return arguments
+
+
+def take_images_from_bucket(search_request: str, source_bucket: str, 
+                            date: datetime, client: Minio) -> List[str]:
+    
+    objects = []
+    prefix = (
+        f'year={date.year}/month={date.month}/'
+        f'day={date.day}/{search_request}/'
+    )
+
+    for obj in client.list_objects(bucket_name=source_bucket, prefix=prefix):
+        objects.append(obj.object_name)
+
+    return objects
+
+
+def resize_image(client: Minio, bucket_object: str,
+                 source_bucket: str, 
+                 width: int, height: int) -> BytesIO:
+
+    file = BytesIO(client.get_object(source_bucket, bucket_object).read())
```

Comment:
Check the examples here and try to see what you haven't done.

https://docs.min.io/docs/python-client-api-reference.html#get_object